lib/helpers.js

```diff
@@ -24,22 +21,24 @@ const getFormatOptions = () => PRETTIER_CORE_OPTIONS.reduce(setConfigOption, {})
 
 const getCurrentScope = () => atom.workspace.getActiveTextEditor().getGrammar().scopeName;
 
-const executePrettier = (text, formatOptions) => {
+const executePrettier = (text, formatOptions, options = { ignoreErrors: false }) => {
   try {
     return prettier.format(text, formatOptions);
   } catch (e) {
     const message = `prettier-atom: ${e.toString()}`;
```

Comment:
we should use a guard clause instead that way it's clear what code goes with what:

```js
  console.log('Error executing Prettier:', e);

  if (!options.ignoreErrors) {
    const message = `prettier-atom: ${e.toString()}`;
    const detail = e.stack.toString();
    atom.notifications.addError(message, { detail, dismissable: true });
  }
  
  return false;
}
```